runtime/onert/api/python/package/infer/session.py

```diff
@@ -134,6 +108,30 @@ def infer(
         # Return the output buffers.
         return (self.outputs.copy(), metrics) if measure else self.outputs.copy()
 
+    def _update_inputs_tensorinfo(self, new_infos: List[tensorinfo]) -> None:
```

Comment:
@ragmani Is there reason to move the location of `_update_inputs_tensorinfo`? It makes me find the differences by scrolling up and down several times. If it moves by some reason and there is no change, could you please leave a comment. It is just a location change.

Reply:
Sorry for not mentioning this earlier. There’s no differences. The function was moved because it was changed from public to private for better code organization.